From 4263060113ac583ec88f7e3a8d6112fa46560d8e Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Thu, 1 Jun 2017 17:18:13 -0700
Subject: [PATCH] MANTA-3291 minnow needs to use new moray, cueball MANTA-3292
 minnow should set cueball pool "maximum" higher MANTA-3090 minnow exits when
 no moray backends are around MANTA-3151 minnow should use more than one moray
 connection

---
 etc/config.coal.json           | 4 ++--
 package.json                   | 2 +-
 sapi_manifests/minnow/template | 4 ++--
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/etc/config.coal.json b/etc/config.coal.json
index b1ac444..1cdc053 100644
--- a/etc/config.coal.json
+++ b/etc/config.coal.json
@@ -16,8 +16,8 @@
         "morayConfig": {
             "srvDomain": "1.moray.coal.joyent.us",
             "cueballOptions": {
-                    "target": 1,
-                    "maximum": 1,
+                    "target": 2,
+                    "maximum": 4,
                     "resolvers": [ "nameservice.coal.joyent.us" ]
             }
         }
diff --git a/package.json b/package.json
index 9ecd65a..119e45d 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
         "cmdutil": "1.1.0",
         "dashdash": "1.3.2",
         "libmanta": "git+ssh://git@github.com:joyent/node-libmanta.git#master",
-        "moray": "3.0.0",
+        "moray": "3.2.0",
         "node-uuid": "1.4.0",
         "once": "1.2.0",
         "statvfs": "2.0.0",
diff --git a/sapi_manifests/minnow/template b/sapi_manifests/minnow/template
index 763fd7f..98e504a 100644
--- a/sapi_manifests/minnow/template
+++ b/sapi_manifests/minnow/template
@@ -16,8 +16,8 @@
                 "morayConfig": {
                         "srvDomain": "{{STORAGE_MORAY_SHARD}}",
                         "cueballOptions": {
-                                "target": 1,
-                                "maximum": 1,
+                                "target": 2,
+                                "maximum": 4,
                                 "resolvers": [ "nameservice.{{DOMAIN_NAME}}" ]
                         }
                 }
-- 
2.21.0

